_data/projects/GlPortal.yml

```diff
@@ -0,0 +1,13 @@
+name: glportal
+desc: A free and open game and game engine with portals.
+site: https://github.com/GlPortal/glPortal
+tags:
+- c++
+- python
+- javascript
+- Game
+- engine
+- 3d
+upforgrabs:
+  name: up-for-grabs
+  link: https://bugs.glportal.de/
```

Comment:
It'd be great if this URL pointed to a filtered list of issues which new contributors might be able to get started on. Is this possible?